pkg/controller/instance/controller_reconcile.go

```diff
@@ -273,6 +273,8 @@ func (igr *instanceGraphReconciler) updateResource(
 	// TODO: Handle annotations
 	desired.SetResourceVersion(observed.GetResourceVersion())
 	desired.SetFinalizers(observed.GetFinalizers())
+	desired.SetOwnerReferences(observed.GetOwnerReferences())
```

Comment:
Are you sure you can flat copy the observed into the desired? Has a merge already happened?